### PR TITLE
Created new qs_gp source, ran makefile

### LINKED ISSUE #63: Quattroshapes: qs vs qs_gp
Once https://github.com/whosonfirst-data/whosonfirst-data/issues/107 lands, update the `quattroshapes.json` source to include the prefix of `qs_gp` (for the point gazetteer).

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #63, replaces #70.

This PR includes a new source file for the Quattroshapes point gazetteer.

The information in the source json is similar to the [`Quattroshapes.json source`](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/quattroshapes.json), but specific to the point gazetteer dataset. Additional files were updated via the Makefile.

### CODE REVIEW

File: sources/quattroshapes-gp.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": 1108970625,
+	"fullname": "Quattroshapes Point Gazetteer",
+	"name": "quattroshapes_gp",
```
Comment (nvkelso) [inline sources/quattroshapes-gp.json]: `quattroshapes_pg` (p before g)

File: sources/quattroshapes-gp.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	"prefix": "qs_gp",
```
Comment (nvkelso) [inline sources/quattroshapes-gp.json]: `qs_gp` (p before g)

File: sources/README.md [hunk 3]
```diff
@@ -862,6 +862,18 @@ _A gazetteer of non-overlapping, authoritative polygons around a curated list of
 * remarks _https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/quattroshapes_remarks.md_
 * license _https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md_
 
+## Quattroshapes Point Gazetteer
+
+_The Quattroshapes point gazetteer. A gazetteer of non-overlapping, authoritative polygons around a curated list of places._
```
Comment (nvkelso) [inline sources/README.md]: `The Quattroshapes point gazetteer. A  big list of point locations that supplements the polygons gazetteer.`

File: sources/quattroshapes-gp.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+   	"key": "",
+	"url": "http://www.quattroshapes.com/",
+	"license": "https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md",
+	"license_type": "CC BY 2.0",
+	"license_text": "Please include attribution in your app, site, or printed work.",
+	"description": "The Quattroshapes point gazetteer. A gazetteer of non-overlapping, authoritative polygons around a curated list of places."
```
Comment (nvkelso) [inline sources/quattroshapes-gp.json]: `The Quattroshapes point gazetteer. A  big list of point locations that supplements the polygons gazetteer.`

**Comment by nvkelso** [comment]
And also fix the merge conflict.

File: sources/quattroshapes-gp.json (hunk 1 above)
Comment (stepps00) [inline sources/quattroshapes-gp.json]: Taken care of in: https://github.com/whosonfirst/whosonfirst-sources/commit/cee74df632e7e7428fbdeee4fc3f28763720c48d.

File: sources/quattroshapes-gp.json (hunk 2 above)
Comment (stepps00) [inline sources/quattroshapes-gp.json]: Taken care of in: https://github.com/whosonfirst/whosonfirst-sources/commit/cee74df632e7e7428fbdeee4fc3f28763720c48d.

File: sources/quattroshapes-gp.json (hunk 4 above)
Comment (stepps00) [inline sources/quattroshapes-gp.json]: Taken care of in: https://github.com/whosonfirst/whosonfirst-sources/commit/cee74df632e7e7428fbdeee4fc3f28763720c48d.

**Comment by stepps00** [comment]
Closing, new qs_pg source added via: https://github.com/whosonfirst/whosonfirst-sources/commit/cee74df632e7e7428fbdeee4fc3f28763720c48d.

**Pull request closed without merging**